staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2217,7 +2217,7 @@ type VirtualMachineInstanceFileSystemList struct {
 
 // VirtualMachineInstanceFileSystemDisk represents the guest os FS disks
 type VirtualMachineInstanceFileSystemDisk struct {
-	Serial  string `json:"serial"`
+	Serial  string `json:"serial,omitempty"`
 	BusType string `json:"bus-type"`
```

Comment:
Will changing this to `busType` come in another PR?

Reply:
I plan to add it to this one, but first I wanted us to decide if it isn't justified to break api consistency to avoid unnecessary code.

Reply:
Added a new commit with the requested api changes.